### ISSUE TITLE
Project Loading failure

### ISSUE THREAD

**Issue by raymondgggg** [ionide/ionide-vscode-fsharp#1508]
![Screenshot from 2021-03-02 12-34-39](https://user-images.githubusercontent.com/25129953/109711400-affc9480-7b53-11eb-84b9-12cf9eee1a38.png)
Issue Type: <b>Bug</b>

I run Pop!_OS  20.04 and .NET 5.0.2, and starting today every time I open an any F# project that I have opened in the past I get an error on the bottom right saying "project loading failed," along with "fsharp.AddFileToProject not found." On top of all that, I'm not able to use ctrl + click to go to definitions and there is no error highlighting. Not sure what to do hoping anyone could help.

Extension version: 5.3.2
VS Code version: Code 1.53.2 (622cb03f7e070a9670c94bae1a45d78d7181fbd4, 2021-02-11T11:48:44.518Z)
OS version: Linux x64 5.8.0-7642-generic

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD Ryzen 5 3600 6-Core Processor (12 x 2199)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>opengl: enabled_on<br>protected_video_decode: unavailable_off<br>rasterization: disabled_software<br>skia_renderer: enabled_on<br>video_decode: disabled_software<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|1, 1, 1|
|Memory (System)|15.60GB (5.93GB free)|
|Process Argv|--no-sandbox --unity-launch --crash-reporter-id 643362ab-9fb5-4c17-afcf-026acd950ebb|
|Screen Reader|no|
|VM|0%|
|DESKTOP_SESSION|pop|
|XDG_CURRENT_DESKTOP|Unity|
|XDG_SESSION_DESKTOP|pop|
|XDG_SESSION_TYPE|x11|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
pythonvspyt617cf:30253240
vspor879:30202332
vspor708:30202333
vspor363:30204092
vstry244cf:30256637
pythonvsdeb440:30248342
pythonvsded773:30248341
pythonvspyt875:30259475
dockersubsetcf:30265999
pythontbcf:30265426

```

</details>

**Comment by baronfel** [comment]
This is the same root cause as https://github.com/ionide/proj-info/issues/95, unknown what the fix is at the moment.

**Comment by rytido** [comment]
Also looks similar to #1479

**Comment by baronfel** [comment]
This is solved in Ionide 5.4.0, released earlier today.
